data/notebooks/Lab/run_marxan.ipynb

```diff
@@ -917,7 +917,11 @@
    "source": [
     "# Create a new input.dat file with by default values\n",
     "test = DatFile()\n",
+    "<<<<<<< local\n",
     "example = inputDatFile(BLM=0.1, NUMREPS =100, PUVSPRNAME= 'puvsp.dat')\n",
+    "=======\n",
+    "example = inputDatFile(BLM=0.1, NUMREPS =100)\n",
+    ">>>>>>> remote\n",
```

Comment:
this looks like a leftover merge conflict

Reply:
it is, I'm going to try to remove it.